### PR TITLE
Copy migrate frequently setup from C2

### PR DESCRIPTION
**Pull request by adelevie**
Implements "migrate frequently" as described here: https://docs.cloudfoundry.org/devguide/services/migrate-db.html and copied from 18F/C2.

### CODE REVIEW

File: manifest-staging.yml [hunk 1]
```diff
@@ -8,3 +8,4 @@ applications:
   buildpack: https://github.com/ddollar/heroku-buildpack-multi.git
   services:
     - micropurchase-staging-psql
+  command: bundle exec rake cf:on_first_instance db:migrate && bundle exec puma
```
Comment (afeld) [inline manifest-staging.yml]: Are you running Puma? If not, you should change the second command here.

**Comment by afeld** [comment]
Just the one comment – otherwise, looks good! I would try to deploy to a staging environment before merging though.

File: manifest-staging.yml (hunk 1 above)
Comment (adelevie) [inline manifest-staging.yml]: we are totally running with puma!

**Comment by adelevie** [comment]
@afeld Thanks! I'm writing tests for the rake task and for `ServerEnv`.

**Comment by adelevie** [comment]
Tests will be in a subsequent PR.

**Pull request merged**